src/lib/Toast/Toast.svelte

```diff
@@ -120,14 +127,16 @@
   .toast {
     padding: var(--toast-padding, 10px);
     font-size: var(--toast-font-size, 14px);
+    font-family: var(--toast-font-family);
+    font-weight: var(--toast-font-weight);
     height: var(--toast-height, auto);
     border-radius: var(--toast-border-radius, 0px);
     width: var(--toast-width, auto);
-    align-items: var(--toast-align-items, center);
+    align-items: var(--toast-align-items);
     margin: var(--toast-margin, 0px 10px 10px 10px);
     justify-content: var(--toast-justify-content, space-between);
     z-index: var(--toast-z-index, 1000);
-    display: var(--taost-dispay, flex);
+    display: var(--toast-display, flex);
```

Comment:
You have changed this variable, make sure to update this variable in if used anywhere